North-Seattle-College/ad440-winter2021-thursday-repo
# AWS lambda function users userid

## To test:

- Use Postman to test Get, Put, Patch, etc.
- Go to url:  https://mchfd7t6l4.execute-api.us-west-2.amazonaws.com/users/{userid}
- Change `{userid}` in the url to a valid userID such as "1" or "1017"
- Hit "Enter" to see the info for that user displayed on the page

-------------------------------------------------------------------------


DATE | ACTIVITY | TIME
-- | -- | --
03/06 | Researched Lambda functions and implementation of task | 1 hr
03/09 | Implementation (group coding) incl. VS Code and AWS | 3 hr
03/10 | Created Issue for task | 3 hr
03/12 | Worked on implementation | 1.5 hr
03/12 | Created PR # 268 | 0.25 hr
